std/haxe/macro/Printer.hx

```diff
@@ -251,6 +251,7 @@ class Printer {
 
 		var str = t == null ? "#NULL" :
 			(printPackage && t.pack.length > 0 && t.pack[0] != "" ? "package " + t.pack.join(".") + ";\n" : "") +
+			(t.doc != null && t.doc != "" ? "/**\n" + tabString + StringTools.replace(t.doc, "\n", "\n" + tabString) + "\n**/\n" : "") +
```

Comment:
well, i guess not, because i copy-pasted that from the field doc output code, so it's consistent with the field doc behaviour.